### PR TITLE
Preview resize

### PR DESCRIPTION
**Pull request by ywwg**

### CODE REVIEW

File: src/skin/skinloader.cpp [hunk 1]
```diff
@@ -65,6 +65,7 @@ QPixmap SkinLoader::getSkinPreview(const QString& skinName, const QString& schem
         preview.load(getSkinPath(skinName) + "/skin_preview.png");
     }
     if (!preview.isNull()){
+        preview = preview.scaled(QSize(640, 360), Qt::KeepAspectRatio, Qt::SmoothTransformation);
         return preview;
     }
     preview.load(":/images/skin_preview_placeholder.png");
```
Comment (ronso0) [inline src/skin/skinloader.cpp:71]:
```suggestion
    if (preview.isNull()) {
        preview.load(":/images/skin_preview_placeholder.png");
    }
    preview = preview.scaled(QSize(640, 360), Qt::KeepAspectRatio, Qt::SmoothTransformation);
```
..to include the placeholder.
if we then create all previews with a ratio of 16:9 all options below have a fixed position.

**Review by ronso0 — CHANGES_REQUESTED** [review]

**Comment by ronso0** [comment]
we should re-create all skin previews when this is merged. Right now they're blurry already due to odd resizing from minimal skin size to 445 x 250 px.
we can take the new, yet to be made 4k screenshots from the website and scale the down to 1920px
https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/2.2E3.20planning.3A.20https.3A.2F.2Fgithub.2Ecom.2Fmixxxdj.2Fmixxx.2Fprojects.2F2/near/240724451

**Comment by ywwg** [comment]
Yes I like that idea

File: src/skin/skinloader.cpp (hunk 1 above)
Comment (ronso0) [reply src/skin/skinloader.cpp:71]: ping

**Comment by ywwg** [comment]
good to merge?

**Comment by ronso0** [comment]
let's wait for CI..

**Comment by ronso0** [comment]
LGTM, thank you!

**Comment by ronso0** [comment]
Unfotunately the previews are blurry when Mixxx is scaled uo, even when using integer values.
fixed with #3941

**Pull request merged**